package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "test": "mocha",
-    "docs": "jsdoc -c conf.json"
+    "docs": "/usr/local/bin/jsdoc --verbose -c conf.json ./node_modules/docdash"
```

Comment:
This is not going to work on Windows.  What is the reason for this change?  Can it be made platform-neutral?

Reply:
That was a mistake. I'd made that local change to get proper doc generated. The problem I ran into was that the jsdoc in ./node_modules did not properly generate the doc pages. If you (@y-polonsk ) can figure out why it doesn't work properly  that'd be great.

Reply:
"npm run docs" works fine for me.  What output do you get?